core/src/main/scala/org/apache/spark/eventhubs/client/CachedEventHubsReceiver.scala

```diff
@@ -55,10 +55,12 @@ private[client] class CachedEventHubsReceiver private (ehConf: EventHubsConf,
     receiverOptions.setReceiverRuntimeMetricEnabled(false)
     receiverOptions.setIdentifier(
       s"spark-${SparkEnv.get.executorId}-${TaskContext.get.taskAttemptId}")
-    _receiver = client.createReceiverSync(consumerGroup,
-                                          nAndP.partitionId.toString,
-                                          EventPosition.fromSequenceNumber(requestSeqNo).convert,
-                                          receiverOptions)
+    _receiver = client.createEpochReceiverSync(
```

Comment:
>createEpochReceiverSync [](start = 23, length = 23)

By changing from standard receiver to epoch receiver - impact for customers is that:
In earlier model they could be running multiple Jobs - by using $default consumerGroup
With Epoch receivers, those jobs would start seeing failures.
eval, if we can take proactive measure here:
1. announce this CHANGE of behv. loud and clear in release notes.
2. add faq - for ReceiverDisconnectedException with epoch errors.

Reply:
One of the ways to solve this could be - to add this possibility of 1 spark job trumping another - in via traces.

Reply:
this is actually a pretty big issue, we should talk about this more before merging